composer.json

```diff
@@ -8,6 +8,7 @@
     "homepage": "https://github.com/dominikb/composer-license-checker",
     "license": "MIT",
     "type": "library",
+    "version": "2.7.0",
```

Comment:
phpunit was complaining that the version was missing, I reused the one from your latest release.